### ISSUE TITLE
Add EventProcessingPipeline

### ISSUE THREAD

**Issue by JacksonMaxfield** [CouncilDataProject/cdp-backend#16]
## Use Case
*Please provide a use case to help us understand your request in context*

Add the core pipeline to the repo.

## Solution
*Please describe your ideal solution*

Bring over the major components of the existing `cdptools.pipelines.EventGatherPipeline` but as a Prefect `Flow`. All files should be handled with `gcsfs` / `fsspec`.

## Alternatives
*Please describe any alternatives you've considered, even if you've dismissed them*

**Comment by JacksonMaxfield** [comment]
Add more documentation:

This relies on #10 as the input to this entire pipeline is the output of some user defined `get_events` function.

i.e.
```python
# in the user script
@task
def get_events() -> List[EventData]:
    ...

# in our pipeline
from user_script import get_events

with Flow("event-gather-pipeline") as flow:
    events = get_events()
    ...
```

**Comment by JacksonMaxfield** [comment]
@Shak2000 when you wrap up on the thumbnail pipeline addition feel free to add in your PR:

"Resolves #16"

**Comment by JacksonMaxfield** [comment]
Resolved in #78
